functional_tests/artifacts/log-model.py

```diff
@@ -0,0 +1,43 @@
+import torch
+import torch.nn as nn
+import torch.nn.functional as F  # noqa
+import wandb
+from wandb.beta.workflows import log_model
+
+
+class Net(nn.Module):
```

Comment:
nit: if it is the same model that being used for all test maybe it can be in a share file... obv pros and cons for both :)

Reply:
Nice -- I think 3 is the magic number. If I use it again in another test, I'l pull it into a shared file :)